CONTRIBUTING.md.template

```diff
@@ -2,8 +2,8 @@ We love pull requests from everyone. By participating in this project, you
 agree to abide by the thoughtbot [code of conduct].
 
 We expect everyone to follow the code of conduct
-in all thoughtbot open source codebases,
-issue trackers, IRC channel, group chat, and mailing lists.
+anywhere in thoughtbot's project codebases,
```

Comment:
In this case, the two are functionally equivalent. It would make it inconsistent with the CoC, so I'm in favor of leaving it as is.